src/react/features/settings/Bar.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import Fullscreen from '../../components/Fullscreen'
+
+const SettingsBar = ({ children, ...props }) => (
```

Comment:
Miksi filun nimi on Bar vaikka komponentti on SettingsBar?

Reply:
Ja jos logiikka on, että featuren nimestä tulee tuo Settings prefix, niin miksi tuossa ylempänä features/search/Result.js filussa on Result eikä SearchResult?

Reply:
Kysymys siis, että:
a) Oliko tarkoituksella vai vahingossa?
b) Oliko tässä muutettu konventiota? Jos oli, niin mikä uusi konventio on?

Reply:
(alempana nimittäin myös komponentin nimi SettingsNavigation ja filun nimi Navigation.js)

Reply:
jep, kokeilin tosiaan uutta konventiota, jossa filun nimessä ei ole featurea. Nopeuttaa uuden feature luontia, mutta muuten taitaa aiheuttaa hämminkiä.

Ehkä voisi lisätä mielummin kaikkiin tiedostonimiin featuren, jolloin actions -> settingsActions etc. reducer -> settingsReducer

Reply:
tehdään tämä eri PR:ssä

Reply:
Tehty täällä: https://github.com/DioryMe/diory-browser-electron/pull/194